pkg/tfvars/openstack/openstack.go

```diff
@@ -22,12 +22,16 @@ type config struct {
 }
 
 // TFVars generates OpenStack-specific Terraform variables.
-func TFVars(masterConfig *v1alpha1.OpenstackProviderSpec, region string, externalNetwork string, lbFloatingIP string, apiVIP string, dnsVIP string, ingressVIP string, trunkSupport string, octaviaSupport string) ([]byte, error) {
+func TFVars(masterConfig *v1alpha1.OpenstackProviderSpec, cloud string, region string, externalNetwork string, lbFloatingIP string, apiVIP string, dnsVIP string, ingressVIP string, trunkSupport string, octaviaSupport string) ([]byte, error) {
+	if cloud == "" {
+		cloud = masterConfig.CloudName
```

Comment:
why is the cloud name on master machine objects different from the the one that should actually be used??

Reply:
Yeah, it's not related here at all... I changed this.
Now, when the installer wants to read a local clouds.yaml file, it looks at "cloud" value from the install-config.yaml first, and, if there is nothing, it tries the default value "openstack".

Reply:
But when we create a system secret with a generated "clouds.yaml" item, it always sets the name as "openstack" there, which is good for consistency.